extensions/gdx-setup/res/com/badlogic/gdx/setup/resources/html/war/index

```diff
@@ -24,6 +24,7 @@
                 evt.preventDefault();
                 evt.stopPropagation();
               }
+              document.addEventListener('contextmenu', event => event.preventDefault());
```

Comment:
What a nice line of code. I don't like seeing context menus from the browser in a game. Approved.